x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts

```diff
@@ -6,18 +6,36 @@
  */
 
 import {
-  EuiMarkdownEditorUiPlugin,
+  EuiLinkAnchorProps,
   getDefaultEuiMarkdownParsingPlugins,
   getDefaultEuiMarkdownProcessingPlugins,
   getDefaultEuiMarkdownUiPlugins,
 } from '@elastic/eui';
-
+// Remove after this issue is resolved: https://github.com/elastic/eui/issues/4688
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Options as Remark2RehypeOptions } from 'mdast-util-to-hast';
+import { FunctionComponent } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import rehype2react from 'rehype-react';
+import { Plugin, PluggableList } from 'unified';
 import * as timelineMarkdownPlugin from './timeline';
-const uiPlugins: EuiMarkdownEditorUiPlugin[] = getDefaultEuiMarkdownUiPlugins();
+
+export const { uiPlugins, parsingPlugins, processingPlugins } = {
+  uiPlugins: getDefaultEuiMarkdownUiPlugins(),
+  parsingPlugins: getDefaultEuiMarkdownParsingPlugins(),
+  processingPlugins: getDefaultEuiMarkdownProcessingPlugins() as [
```

Comment:
Why we need to cast it?

Reply:
there was this but removing now https://github.com/elastic/eui/issues/4688

Reply:
jk https://github.com/elastic/kibana/pull/97646#discussion_r620365295